### DISCUSSION TITLE
Issue in adding a team member

### DISCUSSION THREAD

**Discussion by rbkayz** [supabase/supabase#7767] [Questions]
Hi Community,

Trying to add a dev to our team, but facing an issue here. Could you suggest a solve please?

Thank you

![image](https://user-images.githubusercontent.com/62215539/179543721-d1c8eadc-3347-48cc-92fa-28b314dc36c1.png)

**Comment by silentworks** [comment]
The team member you are inviting will need to create a supabase account first, then they can accept the invite.

**Reply by rbkayz** [reply]
Hi Andrew,

My team member does have an account, and we've tried multiple emails. But the same error is occuring. Any other fix you can suggest?

Thank you

**Reply by hf** [reply]
I know this sounds a bit silly, but maybe clearing all the application data helps? Accept the invitation via an Incognito window?
